src/test/java/com/google/sps/data/PostTest.java

```diff
@@ -471,4 +471,138 @@ public void testInvalidOrgNameLength() {
     Assert.assertFalse(testPost.valid);
   }
 
+  @Test
+  public void testGetShortDuration() {
+
+    String collegeId = "122931";
+    String organizationName = "Bread Club";
+    String month = "12";
+    String day = "31";
+    String startHour = "4";
+    String startMinute = "30";
+    String startAMorPM = "am";
+    String endHour = "4";
+    String endMinute = "31";
+    String endAMorPM = "am";
+    String location = "Benson Memorial Center";
+    String lat = "37.3476132";
+    String lng = "-121.9394005";
+    String numberOfPeopleItFeeds = "3";
+    String typeOfFood = "Bread";
+    String description = "Bread Club is officially giving away free bread!";
+
+    Post testPost = requestToPost(collegeId, organizationName, month, day, startHour, startMinute, startAMorPM, endHour, 
```

Comment:
Is this the expected output from the formatter?

Reply:
Hmm, I'm not sure what you mean by that. Do you mean if a Post object should come out of `requestToPost`?

Reply:
Just chatted with Sanya through Chat and resolved this!